src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -642,7 +642,7 @@ public void addEnvVars(List<TemplateEnvVar> envVars) {
      * Deletes the template.
      */
     @POST
-    public HttpResponse doDoDelete(@AncestorInPath KubernetesCloud kubernetesCloud) throws IOException {
+    public HttpResponse doDoDelete(@AncestorInPath PodTemplateGroup kubernetesCloud) throws IOException {
```

Comment:
Rename variable, since it is no longer a cloud.

```suggestion
    public HttpResponse doDoDelete(@AncestorInPath PodTemplateGroup owner) throws IOException {
```